clusterloader2/testing/density/suite.yaml

```diff
@@ -0,0 +1,4 @@
+- identifier: pod-anti-affinity
```

Comment:
Can we name the file `scheduler-suite.yaml` or something like this?

Other than that it looks good to me

/approve

Reply:
Done.